billing.md

```diff
@@ -99,34 +95,123 @@ Next, add the `Billable` trait to your model definition. This trait provides var
+
+> {note} Passing a Payment Method string directly to the `create()` subscription method will also automatically save it to the user.
+
+#### Payment Methods For Single Charges
+
+When making single charges we'll only need to use a payment method one single time. While you could technically retrieve the default payment method from a customer (which is used for billing and invoicing) and use that to make the payment you can also opt to allow the customer to enter a one-time payment method with the Stripe.js library.
+
+A detailed example can be examined [in the Stripe docs](https://stripe.com/docs/payments/payment-intents/migration#elements). More info about the `createPaymentMethod` call can be found [here](https://stripe.com/docs/stripe-js/reference#stripe-create-payment-method). When you have the `result.paymentMethod.id`, use it to [create single charges](#simple-charge).
+
+<a name="retrieving-payment-methods"></a>
+### Retrieving Payment Methods
+
+The `paymentMethods` method on the billable model instance returns a collection of `Laravel\Cashier\PaymentMethod` instances:
+
+    $paymentMethods = $user->paymentMethods();
+
+To retrieve the default payment method, the `defaultPaymentMethod` method may be used;
+
+    $paymentMethod = $user->defaultPaymentMethod();
+
+<a name="check-for-a-payment-method"></a>
+### Check For A Payment Method
+
+You may check if a customer has a payment method attached to their account using the `hasPaymentMethod` method:
+
+    if ($user->hasPaymentMethod()) {
+        //
+    }
+
+<a name="updating-the-default-payment-method"></a>
+### Updating The Default Payment Method
+
+The `updateDefaultPaymentMethod` method may be used to update a customer's default payment method information. This method accepts a Stripe payment method identifier and will assign the new payment method as the default billing payment method:
+
+    $user->updateDefaultPaymentMethod($paymentMethod);
+
+To sync your default payment method information with the customer's default payment method information in Stripe, you may use the `updateDefaultPaymentMethodFromStripe` method:
+
+    $user->updateDefaultPaymentMethodFromStripe();
+
+> {note} The default payment method on a customer can only be used for invoicing and creating new subscriptions. Because of a limitation by Stripe, it cannot be used for single charges.
+
+<a name="adding-payment-methods"></a>
+### Adding Payment Methods
+
+To add a new payment method, you can call the `addPaymentMethod` method on the billable user:
+
+    $user->addPaymentMethod($paymentMethod);
```

Comment:
What exactly is payment method here? A string? An object? Where do I get it?

Reply:
I've added a link to the "Storing Payment Methods" section for this.